compressors/huffman_coder.py

```diff
@@ -43,6 +44,14 @@ def build_huffman_tree(self) -> HuffmanNode:
         1. Sort the prob distribution, combine last two symbols into a single symbol
         2. Continue until a single symbol is left
         """
+        # For the special case that there is a single symbol, we simply create
```

Comment:
earlier this would just create the code with empty bitarray for this case

Reply:
great find. Although, this was not the reason for Q3 issues right?

Reply:
no, but I got some errors due to this

Reply:
thanks!